packages/contracts-core/contracts/GasOracle.sol

```diff
@@ -84,8 +87,10 @@ contract GasOracle is MessagingBase, GasOracleEvents, InterfaceGasOracle {
     }
 
     /// @notice MVP function to set the summit tip.
-    function setSummitTip(uint256 summitTipWei) external onlyOwner {
-        _summitTipWei = summitTipWei;
+    function setSummitTip(uint256 summitTipWei_) external onlyOwner {
+        if (summitTipWei_ > MAX_SUMMIT_TIP) revert SummitTipTooHigh();
+        summitTipWei = summitTipWei_;
+        // TODO: emit event
```

Comment:
Do we want to add an event now?

Reply:
@aureliusbtc I'm going to add the missing events in the PR that fixes ToB-20 later on